features/minion/core/impl/pom.xml

```diff
@@ -22,9 +22,32 @@
                         <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Version>${project.version}</Bundle-Version>
+                        <Export-Package>org.opennms.minion.core.impl.*;version="${project.version}"</Export-Package>
```

Comment:
If something from the impl-package needs be exported, it should NOT be in the impl package.